components/report-manager-minio-gateway/habitat/hooks/run

```diff
@@ -0,0 +1,15 @@
+#!{{pkgPathFor "core/bash"}}/bin/bash
```

Comment:
This is a basic setup script and with hard-coded credentials to unblock the actual development.
Production ready script will be added as part of issue#5787